src/modules/70-pipeline/y1/components/StepPanel/types.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { IconName } from '@harness/icons'
+
+export enum StepPanelCategory {
+  AddStep = 'Add_Step',
+  AddStepGroup = 'Add_Step_Group',
+  UseStepTemplate = 'Use_Step_Template',
```

Comment:
The values can be in pascal case instead of pascal and snake case
```suggestion
  AddStep = 'AddStep',
  AddStepGroup = 'AddStepGroup',
  UseStepTemplate = 'UseStepTemplate',
```